src/System.Device.Gpio/System/Device/Pwm/Channels/Windows10PwmChannel.Windows.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Threading;
+
+namespace System.Device.Pwm
+{
+    internal class Windows10PwmChannel : PwmChannel
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Windows10PwmChannel"/> class.
+        /// </summary>
+        /// <param name="pwmChip">The PWM chip number.</param>
+        /// <param name="pwmChannel">The PWM channel number.</param>
+        /// <param name="frequency">The frequency in hertz.</param>
+        /// <param name="dutyCyclePercentage">The duty cycle percentage represented as a value between 0.0 and 1.0.</param>
+        /// <param name="polarity">The polarity the channel should start in.</param>
+        public Windows10PwmChannel(
+            int pwmChip,
+            int pwmChannel,
+            double frequency,
+            double dutyCyclePercentage = 0.5,
+            PwmPolarity polarity = PwmPolarity.ActiveHigh)
```

Comment:
i feel like this property is useless, it's less confusing to use `1 - DutyCycle`